docs/versioned_docs/version-v7.8.x/03-light-clients/03-localhost/03-client-state.md

```diff
@@ -33,8 +33,6 @@ It is possible to disable the localhost client by removing the `09-localhost` en
 
 The latest height is updated periodically through the ABCI [`BeginBlock`](https://docs.cosmos.network/v0.47/building-modules/beginblock-endblock) interface of the 02-client submodule in core IBC.
 
-[See `BeginBlocker` in abci.go.](https://github.com/cosmos/ibc-go/blob/09-localhost/modules/core/02-client/abci.go#L12)
```

Comment:
Can we change the link to https://github.com/cosmos/ibc-go/blob/v7.8.0/modules/core/02-client/abci.go#L12